src/Storages/ColumnsDescription.h

```diff
@@ -84,18 +84,21 @@ class ColumnsDescription
         Ordinary = 1,
         Materialized = 2,
         Aliases = 4,
+	Ephemeral = 8,
```

Comment:
[Style check](https://s3.amazonaws.com/clickhouse-test-reports/34424/465dee928cc0be24d9889fc975ce477754794124/style_check__actions_/style_output.txt) tells here is a tab instead of spaces.